extensions/pom.xml

```diff
@@ -2083,14 +2083,17 @@
     <dependency>
       <groupId>com.hitachivantara.security.web</groupId>
       <artifactId>security-web-server-api</artifactId>
+      <version>0.9.0</version>
```

Comment:
Version should be changed in the global pentaho parent pom:
https://github.com/pentaho/maven-parent-poms/blob/4a6d77cda21945af9aa382f177ccd7eab53507c1/pom.xml#L373